lib/util/emitDeprecation.js

```diff
@@ -0,0 +1,20 @@
+const warningMessages = {
+    CREATE_CHANNEL_OPTIONS: "createChannel() was called with a string `options` argument",
+    CREATE_CHANNEL_REASON: "createChannel() was called with a string `reason` argument",
```

Comment:
These two messages are basically the same, and both have the same root cause - the signature of `createChannel` has changed. Should we merge them into the same code?